src/detectors/RPOTS/RomanPotsReconstruction_factory.cc

```diff
@@ -19,8 +19,14 @@ namespace eicrecon {
 
 	m_log = app->GetService<Log_service>()->logger("ForwardRomanPotRecParticles");
 
-	/*
-        auto id_spec = detector->readout(m_readout).idSpec();
+	m_readout = "ForwardRomanPotHits";
+	app->SetDefaultParameter("RPOTS:ForwardRomanPotHits:readoutClass", m_readout);
```

Comment:
It is better to use PluginName() function instead of directly writing "RPOTS". And also output tag instead of ForwardRomanPotHits

Here is an exmple: 
https://github.com/eic/EICrecon/blob/main/src/global/tracking/TrackParameters_factory.cc#L21

Reply:
Okay, I fixed this. I am still not sure about the "readoutClass" part of this, so I have that written in by hand for the moment, but I did use the same setup as in the example for the first two pieces of the string.